AutoRest/Generators/Ruby/Azure.Ruby.Tests/RspecTests/paging_spec.rb

```diff
@@ -19,102 +19,122 @@
 
   # Paging happy path tests
   it 'should get single pages' do
-    result = @client.paging.get_single_pages_async().value!
-    expect(result.response.status).to eq(200)
-    expect(result.body.next_link).to be_nil
+    all_items = @client.paging.get_single_pages()

```

Comment:
should we be also verifying the type of the result (all_items is of type page ) ?